backend/graphql/resolvers/schoolResolvers.ts

```diff
@@ -24,18 +30,6 @@ const schoolResolvers = {
       const newSchool = await schoolService.createSchool(school);
       return newSchool;
     },
-    addTeacherToSchool: async (
```

Comment:
i'm wondering if there's still a separate use case for `createSchool` and `addTeacherToSchool` if i directly implement these in `registerTeacher`? @joyce-shi

Reply:
LOL wait, I feel like it's so sad to delete. But I can't think of any separate use case? I guess if we need them we can always come back to the commit history and find them again D: